src/Mod/Path/Gui/Resources/Path.qrc

```diff
@@ -61,9 +61,11 @@
         <file>icons/Path-Speed.svg</file>
         <file>icons/Path-Stock.svg</file>
         <file>icons/Path-Stop.svg</file>
+        <file>icons/Path-ThreadMilling.svg</file>
         <file>icons/Path-ToolBit.svg</file>
         <file>icons/Path-ToolChange.svg</file>
         <file>icons/Path-ToolController.svg</file>
+	      <file>icons/Path-ToolDuplicate.svg</file>
```

Comment:
fix whitespace here

Reply:
I don't even know where this comes from - must be some leprechaun messing with my sources....

Reply:
@mlampert do you mind using an underscore for the SVG instead of dash? (due to #4095)
i.e. `Path_ToolDuplicate.svg`